js/interface.js

```diff
@@ -2537,7 +2537,8 @@ $('[data-save-rule]').click(function(event) {
     rule.require = requiredFields;
   }
 
-  var exclude = _.compact($('input[name="exclude"]').val().split(','));
+  // Spilt exclude columns list with ', ' as it contain space between columns names
+  var exclude = _.compact($('input[name="exclude"]').val().split(',').map(s=>s.trim()));
```

Comment:
Doesn't seem to be in line with ESLint. I am expecting to see `s => s.trim()`

Can you also name `s` accordingly to its context? I would suggest `column`

Reply:
ok i will check and add accordingly